build-tools/mono-runtimes/mono-runtimes.projitems

```diff
@@ -1,7 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+      Each _MonoRuntime is placed in its own ItemGroup to keep it together with the associated PropertyGroup that
```

Comment:
I agree in principle, but I thought this mixing is justified here because the properties serve a very specific and narrow purpose, tightly tied to the item definition that follows. Putting them in a separate file would create unnecessary confusion and force whoever works with those files in the future to hunt for the definitions of the properties somewhere else - inefficient.